.travis.yml

```diff
@@ -60,6 +60,13 @@ matrix:
         - TESTMODE=full
         - COVERAGE=--coverage
         - NUMPYSPEC=numpy
+  allow_failures:
+    - python: 2.7
+      env:
+        - TESTMODE=fast
+        - COVERAGE=
+        - NUMPYSPEC="--upgrade git+git://github.com/numpy/numpy.git@master"
+
```

Comment:
Ah, yes, you are right.  Should have checked the 'travis-dev-wheels' part before I made this change.